examples/memory_basic_usage.py

```diff
@@ -0,0 +1,138 @@
+"""
+========================
+How to use joblib.Memory
+========================
+
+This example illustrates the usage of :class:`joblib.Memory`. We illustrate
+usages with a function and a method.
+
+"""
+
+###############################################################################
+# Without :class:`joblib.Memory`
+###############################################################################
+# 
+# To show the benefit of using :class:`joblib.Memory`, we will implement a
+# function which is computationally expensive to execute. First, we will check
+# the time required to perform the decomposition on a large array of data.
+
+import time
+import numpy as np
+
+
+def expensive_computation(data, column_index=0):
+    """Simulate an expensive computation"""
+    time.sleep(5)
+    return data[column_index]
+
+
+rng = np.random.RandomState(42)
```

Comment:
I think that here a comment on the importance of seeding random number generators for memory to work would be useful.

Reply:
No need for it in fact.